pkgs/top-level/all-packages.nix

```diff
@@ -27989,6 +27989,16 @@ with pkgs;
 
   prusa-slicer = callPackage ../applications/misc/prusa-slicer { };
 
+  snapmaker-luban = callPackage ../applications/misc/snapmaker-luban {
+    branch = "stable";
+    inherit pkgs;
```

Comment:
You're using callPackage so you don't need to `inherit pkgs;`

Reply:
Fixed